files/galaxy/dynamic_rules/usegalaxy/total_perspective_vortex/tools.yml

```diff
@@ -42,6 +42,26 @@ tools:
       require:
         - singularity
 
+
+  basic_docker_tool:
+    scheduling:
+      require:
+        - docker
+  basic_illumination:
```

Comment:
Cores and mem should be inherited from default tool?

Reply:
for a first go yes, I do not know the number of cores yet